llvm/test/Transforms/GVN/PRE/no-phi-translate.ll

```diff
@@ -0,0 +1,57 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -passes=gvn %s | FileCheck %s
+
+; NOTE: when we use a Token like type we should not introduce a phi
```

Comment:
@bogner i did put this note in

Reply:
This note makes sense - I was saying it might also be good to note that in this case we expect GVN not to do anything at all, which gives some context for reading what the check lines are checking.

So maybe something like "Test that GVN doesn't introduce phis for token like types. NOTE: This implies that GVN won't do anything at all to this input". OTOH maybe the existing comment is clear enough. I'll leave it to you to decide.